lib/importo/import_column.rb

```diff
@@ -59,5 +59,9 @@ def overridable?
     def collection
       options[:collection]
     end
+
+    def delay
```

Comment:
Best to document this column in the README

Reply:
done